gen/definitions/profile_parcels/system_remote_access.yaml

```diff
@@ -15,34 +15,44 @@ attributes:
     example: false
   - model_name: anyConnectEapAuth
     tf_name: any_connect_eap_authentication_type
-    example: user
-  - model_name: anyConnectProfileDownloadStatus
-    tf_name: any_connect_eap_profile_download_status
-    conditional_attribute:
-      name: anyConnectEapAuth
-      value: device
-    example: INITIALIZED
-  - model_name: anyConnectProfileFileName
-    tf_name: any_connect_eap_profile_file_name
     conditional_attribute:
-      name: anyConnectEapAuth
-      value: device
-    example: "<form-data>"
+      name: connection_type_ssl
+      value: false
+      type: Bool
+    example: user
+    # ==== Requires support for file upload ===
+  # - model_name: anyConnectProfileDownloadStatus
+  #   tf_name: any_connect_eap_profile_download_status
+  #   example: INITIALIZED
+      # conditional_attribute:
+      #     name: any_connect_eap_authentication_type
+      #     value: device
+  # - model_name: anyConnectProfileFileName
+  #   tf_name: any_connect_eap_profile_file_name
+  #   example: "<form-data>"
+      # conditional_attribute:
+      #     name: any_connect_eap_authentication_type
+      #     value: device
   - model_name: ipv4PoolSize
     example: 50
   - model_name: ipv6PoolSize
     example: 1024
   - model_name: enableCrlCheck
+    tf_name: enable_certificate_list_check
```

Comment:
If we add/remove/change attributes of an existing resource we should list those breaking changes in the changelog.